src/EventListener/InitializeSystemListener.php

```diff
@@ -181,10 +190,7 @@ protected function boot(Request $request = null)
         $this->initializeLegacySessionAccess();
         $this->setDefaultLanguage();
 
-        // Fully load the configuration
-        $objConfig = Config::getInstance();
```

Comment:
In this case, it does make a huge difference when `preload()` and `instantiate()` are called. If the adapter changes the program flow, I cannot merge it, because we cannot foresee the ramifications.